content/blog/2023/when-to-choose-a-responsive-web-app.md

```diff
@@ -0,0 +1,160 @@
+---
+title: When to Choose a Responsive Web App
+sub: Part 1 - Responsive Web App vs PWA vs Native Mobile App
+author: sondra
+date: 2023-05-05
+tags:
+  - Article
+  - Business Development
+  - Push Notifications
+  - Recommendations
+  - Startups
+  - User Experience
+image:
+  src: blog/2023/part1-web.jpg
+  alt: |
+    spider web with dew drops
+summary: |
+  The decision of what platform
+  to build your app with
+  is quite important –
+  affecting project scope, timeline, and budget.
+  But understanding the differences between
+  a responsive web app,
+  a progressive web app,
+  and a native mobile app,
+  and deciding which one is right for your project
+  can seem insurmountably difficult.
+  Welcome to part 1 of a three-part series
+  unpacking the reasons to choose
+  one platform over another.
+  Let's start with the responsive web app (RWA).
+---
+{% import 'embed.macros.njk' as embed %}
+
+You have an idea for a custom digital product you’d like to build, and it’s time to decide how to build it. One of your friends swears you won’t succeed unless you build a native mobile app for iPhone. A colleague argues that a responsive web app is the only way to go. Who’s right? Are there alternative options? The decision of how to build your app is quite important – affecting project scope, timeline, and budget – but understanding when to choose one option over another can be overwhelming.
+
+The great news is that you don’t need to spend years becoming a web developer yourself in order to make a wise choice. In the following three-part series, we’ll dig into the three main options for building mobile apps: Responsive Web Apps, Progressive Web Apps, and Native Mobile Apps, and how to determine the best option for your project. As you read, think about who your app will serve, what their most pressing needs are, and how your app will meet those needs.
+
+**Read all three parts to compare app types:**
+
+
+1. [When to Choose a Responsive Web App](/2023/05/05/when-to-choose-a-responsive-web-app/)
+2. [When to Choose a Native Mobile App](/2023/06/07/when-to-choose-a-native-mobile-app/)
+3. [When to Choose a Progressive Web App](/2023/07/05/when-to-choose-a-progressive-web-app/)
+
+
+## What are Responsive Web Apps, PWAs, and Native Mobile Apps?
+Here are the biggest differences in a nutshell.
+
+- **Responsive Web App (RWA)** - An application that runs in a browser, such as Chrome, Firefox, or Safari, and responds to fit any screen size on any device.
+- **Progressive Web App (PWA)** - A responsive web app with additional capabilities that mimic a native mobile app.
+- **Native Mobile App** - An app built specifically for one particular mobile operating system, like iOS or Android.
+
+
+{{ embed.img(
+  src='blog/2023/comparison-chart.jpg',
+  alt='A comparison chart showing the differences between
+  responsive web apps, progressive web apps, and native
+  mobile apps'
+) }}
```

Comment:
This leaves all the details of the chart pretty inaccessible. Could we do the comparison in HTML, rather than in an image?

(also a little skeptical about 'best capabilities' as a heading. Most maybe? Or most device-specific capabilities?)

Reply:
@mirisuzanne @stacyk Stacy and I discussed releasing this comparison chart as an image temporarily and then building it in HTML. This article is intended to help promote newsletter sign up, so we didn't want to have to delay publication while waiting for the chart to be built out.

Reply:
@mirisuzanne @stacyk 
I added a summary of the comparison chart. I don't assume I've done it in the right place, but the content is here: /rwa-pwa-native-comparison-chart-summary/ however you all recommend.